vmdb/app/models/vim_performance_state.rb

```diff
@@ -82,27 +82,27 @@ def host_count_off
 
   def storages
     ids = get_assoc(:storages, :on)
-    return ids.empty? ? [] : Storage.find_all_by_id(ids, :order => :id)
+    return ids.empty? ? [] : Storage.where(:id => ids).order(:id).to_a
```

Comment:
Note: @tenderlove mentioned that he thinks some of these methods are being accidentally chained but it's not clear why tests are failing on rails4.2 without these .to_a.  I say we move on, trying to keep .to_a's out except where we're blocked by them and come back to them later.